e2e/tests/api_tests/sdo_R2/sdo_R2_test.js

```diff
@@ -0,0 +1,24 @@
+const config = require('../../../config.js');
+
+const mpScenario = 'ONE_V_ONE_FLIGHT_DELAY';
+const claimAmountJudge = '11000';
+
+Feature('CCD 1v1 API test flight delay claim @api-tests-1v1 @api-nonprod');
+
+async function prepareClaim(api_spec, claimAmount) {
+  await api_spec.createClaimSpecFlightDelay(config.applicantSolicitorUser, mpScenario, claimAmount);
+  await api_spec.informAgreedExtensionDate(config.applicantSolicitorUser);
+  await api_spec.defendantResponse(config.defendantSolicitorUser);
+  await api_spec.claimantResponse(config.applicantSolicitorUser, 'FULL_DEFENCE', 'ONE_V_ONE', 'AWAITING_APPLICANT_INTENTION');
+}
+
+Scenario.only('1v1 specified - flight delay claim', async ({api_spec}) => {
```

Comment:
.only is not required.
We also need to check for the "isSdoR2Enabled" flag before launching this test.